maven-core/pom.xml

```diff
@@ -235,11 +235,16 @@ under the License.
             <goals>
               <goal>java</goal>
               <goal>xpp3-reader</goal>
+              <goal>xpp3-writer</goal>
```

Comment:
do we really need a writer for cache files?

Reply:
ok, I found my answer: not really for domain or config, but we need it for diff and report